src/common/mode/native.rs

```diff
@@ -122,7 +122,7 @@ where
                 }
                 std::cmp::Ordering::Equal => {
                     max_value = match max_value {
-                        Some(ref current_max_value) if value < current_max_value => Some(*value),
+                        Some(ref current_max_value) if value > current_max_value => Some(*value),
```

Comment:
why this change?

Reply:
Previously, we returned the MIN value for native types and the first encountered value for strings when multiple values had the same frequency. Now, MAX is used for both, but we can go with MIN in both cases, by the way.